src/charging-station/Bootstrap.ts

```diff
@@ -154,13 +156,37 @@ export default class Bootstrap {
         poolOptions: {
           workerChoiceStrategy: Configuration.getWorkerPoolStrategy(),
         },
-        messageHandler: async (msg: ChargingStationWorkerMessage) => {
-          if (msg.id === ChargingStationWorkerMessageEvents.STARTED) {
-            this.uiServer.chargingStations.add(msg.data.id as string);
-          } else if (msg.id === ChargingStationWorkerMessageEvents.STOPPED) {
-            this.uiServer.chargingStations.delete(msg.data.id as string);
-          } else if (msg.id === ChargingStationWorkerMessageEvents.PERFORMANCE_STATISTICS) {
-            await this.storage.storePerformanceStatistics(msg.data as unknown as Statistics);
+        messageHandler: (msg: InternalChargingStationWorkerMessage) => {
+          const workerEventStarted = (data: ChargingStationData) => {
```

Comment:
No need to create intermediate variables for arrow function. The whole purpose of arrow function is that they can be used inline in the code.

Reply:
Isn't it clearer that way ? Because without it i would need to write something like:
`((arg1: type) => {
/*
**  dosomething
*/
})(value);`
Inside the switch.

Reply:
Not inline the message handler array function. You factor them out as attributes to the current class to simplify and later on in a dedicated class for all message type handler as static methods.